src/main/java/mate/academy/annotation/FieldMatchValidator.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.annotation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.apache.commons.beanutils.BeanUtils;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch, Object> {
+    private String firstFieldName;
+    private String secondFieldName;
+    private String message;
+
+    @Override
+    public void initialize(FieldMatch constraintAnnotation) {
+        firstFieldName = constraintAnnotation.first();
+        secondFieldName = constraintAnnotation.second();
+        message = constraintAnnotation.message();
+    }
+
+    @Override
+    public boolean isValid(final Object value, final ConstraintValidatorContext context) {
+        boolean valid = true;
+        try {
+            final Object firstObj = BeanUtils.getProperty(value, firstFieldName);
+            final Object secondObj = BeanUtils.getProperty(value, secondFieldName);
+            valid = firstObj == null && secondObj == null
+                    || firstObj != null && firstObj.equals(secondObj);
+        } catch (final Exception ignore) {
+            // ignore
+        }
```

Comment:
what's purpose of try-catch block here?
```suggestion
            final Object firstObj = BeanUtils.getProperty(value, firstFieldName);
            final Object secondObj = BeanUtils.getProperty(value, secondFieldName);
            valid = firstObj == null && secondObj == null
                    || firstObj != null && firstObj.equals(secondObj);
```